.github/workflows/e2e-test.yml

```
@@ -0,0 +1,107 @@
+on: push
+
+name: Rhino Usage Test

```

Let's stick to calling this "E2E Test".

```suggestion
name: E2E Test
```